### PR TITLE
Add Probability Alternative

### PR DESCRIPTION
**Pull request by Cybermise**
Added an alternative Probability course. It covers the same content as the existing one. Some would say that this one is better, the only reason I would recommend this one is that it has a full version outside of EdX, so learners can take their time to understand material if they choose not to pay. This one has many exercices for practice. It also has an EdX version, I tried it and didn't like it that much. **Optional book** not realted to this course but it covers everything literally step-by-step (it saved me in combinatronics): https://www.goodreads.com/en/book/show/29991429-probability

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -215,9 +215,10 @@ Courses | Duration | Effort | Prerequisites
 [Introduction to Logic](https://www.coursera.org/learn/logic-introduction) | 10 weeks | 4-8 hours/week | [set theory](https://www.youtube.com/playlist?list=PL5KkMZvBpo5AH_5GpxMiryJT6Dkj32H6N)
 
 #### Probability
-Courses | Duration | Effort | Prerequisites
-:-- | :--: | :--: | :--:
-[Introduction to Probability - The Science of Uncertainty](https://www.edx.org/course/introduction-probability-science-mitx-6-041x-2) | 18 weeks | 12 hours/week | [Multivariable Calculus](https://ocw.mit.edu/courses/mathematics/18-02sc-multivariable-calculus-fall-2010/index.htm)
+Courses | Duration | Effort | Notes | Prerequisites
+:-- | :--: | :--: | :--: | :--:
+[Introduction to Probability - The Science of Uncertainty](https://www.edx.org/course/introduction-probability-science-mitx-6-041x-2) | 18 weeks | 12 hours/week | - | [Multivariable Calculus](https://ocw.mit.edu/courses/mathematics/18-02sc-multivariable-calculus-fall-2010/index.htm)
+[Alternative: Harvard Stats110](https://projects.iq.harvard.edu/stat110/home) | 18 weeks | 12 hours/week | Prefer classroom vibe? | [Multivariable Calculus](https://ocw.mit.edu/courses/mathematics/18-02sc-multivariable-calculus-fall-2010/)
```
Comment (Cybermise) [inline README.md:221]: I also changed the Multivariable prerequisite for this class to the MIT OCW Multi-Var course.

**Comment by spamegg1** [comment]
Nice find. I think the main issue is still the Multivariable Calculus prerequisite. It's so overkill it's not funny. All one needs is a few iterated integrals. It would be nice if we could find a Probability alternative that handles all the integration required in a self-contained way.

I would greatly appreciate it if someone who goes through the MIT course can write down all the integrals evaluated in the course, so maybe I could write up a minimal PDF document covering just those integrals.

**Comment by Cybermise** [comment]
I'm just starting the Multi-Var course so I'll make sure to send back the integral evaluations in the course.. It's gonna take a long time tho there must be a compilation of these integrals somewhere.

**Comment by spamegg1** [comment]
Hey don't push yourself! No hurry.

**Comment by waciumawanjohi** [comment]
This is a [substantive change](https://ocw.mit.edu/courses/mathematics/18-01sc-single-variable-calculus-fall-2010/), so an issue should be opened to give contributors the chance to comment. Can you write up an RFC and post it in the issues. Including a link to this pull request will be helpful to show others exactly the change you envision.

**Comment by waciumawanjohi** [comment]
Resolved with 405df431aa44e049ee40f9300586a556620cbdbd

**Pull request closed without merging**